djstripe/models.py

```diff
@@ -189,15 +193,17 @@ def get_or_create(cls, subscriber, livemode=djstripe_settings.STRIPE_LIVE_MODE):
         try:
             return Customer.objects.get(subscriber=subscriber, livemode=livemode), False
         except Customer.DoesNotExist:
-            return cls.create(subscriber), True
+            action = "create:%s" % (subscriber.pk)
```

Comment:
My gut reaction is that this should be pulled up, but it totally makes sense that this need to be tied very closely to each action.